lib/components/screens/likes_retweets/retweets/cubit/retweets_cubit.dart

```diff
@@ -0,0 +1,116 @@
+import 'dart:convert';
+
+import 'package:bloc/bloc.dart';
+import 'package:built_collection/built_collection.dart';
+import 'package:dart_twitter_api/twitter_api.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:harpy/api/api.dart';
+import 'package:harpy/components/components.dart';
+import 'package:harpy/core/core.dart';
+
+part 'retweets_cubit.freezed.dart';
+
+class RetweetsCubit extends Cubit<PaginatedState<RetweetedUsersData>>
+    with RequestLock, HarpyLogger {
+  RetweetsCubit({required this.tweetId})
+      : super(const PaginatedState.loading());
+  late String tweetId;
+  UserSortBy sort = UserSortBy.fromJsonString(
```

Comment:
```suggestion
  RetweetsCubit({
    required this.tweetId,
  }) : super(const PaginatedState.loading());

  final String tweetId;

  UserSortBy sort = UserSortBy.fromJsonString(
```

`tweetId` should be `final`, the formatting changes are optional